src/utils/facilities/formatCostCenterToServer.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Normalizes the cost center object being sent to the API server
+ *
+ * @param {CostCenter} input
+ *
+ * @returns {Object} output
+ * @returns {string} output.description
+ * @returns {string} output.name
+ * @returns {string} output.organization_id UUID
```

Comment:
Should these be `input.` instead of `output.`?

Reply:
it's a return, so I think output is appropriate

Reply:
Do we do this on other formatters or do we always call it `input` regardless?

Reply:
Correct, `ToServer`s are returns and return outputs, and `FromServer`s have params with inputs across the board.